crypto/x509/x509_vfy.c

```diff
@@ -2910,6 +2945,13 @@ static int build_chain(X509_STORE_CTX *ctx)
         return 0;
     }
 
+    ss = cert_self_signed(ctx, cert);
+    if (ss == -1) {
```

Comment:
In other places, you test `ss < 0`...  which is what is usually done in OpenSSL code.  Whichever way you go, I would like to ask you to be consistent.

(the "norm" seems to be `x < 0` to check for underlying / unsupported type of errors, `x <= 0` to check for all errors and `x > 0` for success)

Reply:
Fixed